lib/export.js

```diff
@@ -165,6 +165,22 @@ class FHIRExporter {
           ssEl.mapping.forEach(fixURI);
         }
       }
+
+      // Insert mapping rules into the FHIR mapping element
+      for (let r of map.rules) {
+        if (r.sourcePath == undefined) {
+          break;
+        }
+        const rulePath = `${map.targetItem}.${r.target}`;
+        // Find the profile snapshot element that corresponds to the mapping rule
+        const elem = profile.snapshot.element.find(e => e.id == rulePath);
+        // If the snapshot element was found, add the mapping in as an 'shr' identity
+        // NOTE: For the time being, this only adds the mapping for the toFHIR test cases, to simplify code
+        if (elem && !r.sourcePath[0].isPrimitive) {
+          elem.mapping.push({ 'identity': 'shr', 'map': `<${r.sourcePath[0].fqn}>` });
+        }
+      }
+      
       // (2) Fix incorrect display text for US jurisdiction
```

Comment:
You are in between (1) and (2) of FHIR STU3 fixes, which is strange.  Either move up or down one code block.